src/components/FlowRunStateTypeTabs.vue

```diff
@@ -54,19 +56,19 @@
     return tabStates[tab]
   }
 
-  function getStateTypeFilter(tab: string): FlowRunsFilter {
-    const types = getTabStates(tab)
-
-    return {
-      ...props.filter,
-      flowRuns: {
-        ...props.filter?.flowRuns,
-        state: {
-          ...props.filter?.flowRuns?.state,
-          type: types,
+  function getStateTypeFilterGetter(tab: string): Getter<FlowRunsFilter> {
```

Comment:
And in direct conflict with my other comment, I think this name change is fine since it's describing the return value lol

Reply:
Lol same cause it returns a getter.